run/websockets/package.json

```diff
@@ -12,12 +12,12 @@
   "dependencies": {
     "@socket.io/redis-adapter": "^7.0.0",
     "express": "^4.17.1",
-    "redis": "^4.0.0",
+    "redis": "^3.1.2",
```

Comment:
Is there a way to avoid downgrading this?

Reply:
There is a breaking change in the library. To avoid rewriting the entire sample this is needed.

Reply:
Got it. Is this something we plan to fix separately?

Reply:
https://github.com/GoogleCloudPlatform/nodejs-docs-samples/issues/2657